.gitignore

```diff
@@ -8,3 +8,4 @@ build/
 packages
 .idea/
 
+pubspec.lock
```

Comment:
I'll stick with keeping `pubspec.lock` otherwise I see no point in tagging my repos when I can't restore the state including the dependencies.
I want to know what dependency versions where available/used at a point in time.
Thanks again for the PR :-)